src/test/resources/fixtures/item.csv

```diff
@@ -0,0 +1,2 @@
+text,fields,register
+"The Entry 1",field1,value1

```

Comment:
do we need a test for an item containing multiple `fields`?  Previously we had a test to ensure we could see that JSON `"fields":["field1","field2"]` was rendered as a CSV column `field1;field2`; this was in the `/records` resource test which might explain why we've lost it here.